test/components/HypernovaExample.js

```diff
@@ -1,6 +1,6 @@
 const hypernova = require('../../');
 
 module.exports = hypernova({
-  server: function () {},
-  client: function () {},
+  server() {},
```

Comment:
Tests are now failing, because node 0.12 doesn't support the modern syntax you're using. I think the only change that can stay is in the example.